package.json

```diff
@@ -4,12 +4,14 @@
   "description": "the Istanbul command line interface",
   "main": "index.js",
   "scripts": {
-    "pretest": "npm run clean && npm run instrument",
-    "test": "tap -t360 --no-cov -b ./test/*.js && npm run report",
-    "posttest": "standard",
+    "lint": "standard",
+    "pretest": "npm run lint && npm run clean && npm run instrument",
+    "test": "tap -t360 --no-cov -b test/*.js",
+    "resnap": "cross-env TAP_SNAPSHOT=1 tap -t360 --no-cov -b test/nyc-integration-snap.js",
```

Comment:
The more common convention is to call this script `"snap"`.  (In tap v13, if the script is named "snap", then it regenerates snapshots by default.)

Also, why just that one file?  Snapshots are useful whenever output is being tested, may as well just run the whole suite.

Reply:
I was just running the one file because I was regenerating the snap file many, many times.  Right now nyc-integration-snap.js is the only source which uses snapshots, I didn't want to go too far without feedback.

One question, in test/nyc-integration-snap.js I have:
```js
if (process.env.TAP_SNAPSHOT !== '1') {
  t.jobs = os.cpus().length
}
```

Obviously setting `t.jobs` is set to enable running multiple tests at a time.  I disabled this when snapshots are being generated to force a stable order.  Is this something tap could improve on?  Either automatically locking `t.jobs == 1` when snapshots are being generated or otherwise do something to enforce a stable order of the snapshot file?  Possibly would use a lot of memory to buffer all snapshots until they can be written in order so maybe just ignore attempts to enable parallelism?

Reply:
The memory overhead is pretty minimal.  But yeah, forcing `t.jobs=1` is good when snapshotting, or else you get a lot of churn in the snapshot file.  But better yet, it should probably just sort the snapshot file before writing it.